swagger.json

```diff
@@ -1821,6 +1821,57 @@
             }
           }
         },
+        "/fleet/vehicles/{vehicle_id}/stats": {
```

Comment:
Nit: the current format is {vehicle_id:[0:9]+} for the other endpoints

Reply:
I have `{vehicle_id:[0:9]+} ` in the `summary` for the `stats` endpoint that I'm adding. It doesn't look like we add the regex in the list of routes here, see L1765.